xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/TestUtils.java

```diff
@@ -1427,6 +1432,46 @@ public String getSecretToken()
         return this.secretToken;
     }
 
+    /**
+     * Get the WCAG context for the test suite.
+     *
+     * @return WCAG Context
+     */
+    public WCAGContext getWcagContext()
+    {
+        return this.wcagContext;
+    }
+
+    public void writeWCAGResults(Logger logger) throws IOException{
+        WCAGContext wcagContext = getWcagContext();
+        if (wcagContext.isWcagEnabled()) {
+            float totalTime = (float) wcagContext.wcagTimer / 1000;
+            logger.info("Time spent on WCAG validation [{}] (in s)", totalTime);
+            File wcagDir = new File("target/wcag");
```

Comment:
For next commit, I set this value as a static final variable in TestUtils (and built it using `File.separator` instead of just a `/` ).
This should be better but I'm not sure this solves this issue completely, I'm not sure of what the best practice here would be.

Reply:
I've just had a look and found https://github.com/xwiki/xwiki-platform/blob/e44c811a38d395a58722110e914ee8a1e6dfb00c/xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/internal/junit5/DockerTestUtils.java#L273 and https://github.com/xwiki/xwiki-platform/blob/a25ac4c3dfacd781d6ed1a86d40ca517a7da26e6/xwiki-platform-core/pom.xml#L340 used at https://github.com/xwiki/xwiki-platform/blob/8793d8438ebfb7cfe40d9b4ce31d9f791f563187/xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/TestDebugger.java#L63 - I think you could/should do something similar for the WCAG results.